weaviate/proto/v1/tenants_pb2_grpc.py

```diff
@@ -1,3 +1,30 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
 """Client and server classes corresponding to protobuf-defined services."""
 import grpc
+import warnings
+
+
+GRPC_GENERATED_VERSION = "1.64.1"
+GRPC_VERSION = grpc.__version__
+EXPECTED_ERROR_RELEASE = "1.65.0"
+SCHEDULED_RELEASE_DATE = "June 25, 2024"
+_version_not_supported = False
+
+try:
+    from grpc._utilities import first_version_is_lower
+
+    _version_not_supported = first_version_is_lower(GRPC_VERSION, GRPC_GENERATED_VERSION)
+except ImportError:
+    _version_not_supported = True
+
+if _version_not_supported:
+    warnings.warn(
+        f"The grpc package installed is at version {GRPC_VERSION},"
+        + f" but the generated code in v1/tenants_pb2_grpc.py depends on"
+        + f" grpcio>={GRPC_GENERATED_VERSION}."
+        + f" Please upgrade your grpc module to grpcio>={GRPC_GENERATED_VERSION}"
+        + f" or downgrade your generated code using grpcio-tools<={GRPC_VERSION}."
+        + f" This warning will become an error in {EXPECTED_ERROR_RELEASE},"
+        + f" scheduled for release on {SCHEDULED_RELEASE_DATE}.",
+        RuntimeWarning,
+    )
```

Comment:
Was this auto-generated? If not, will it get clobbered when `regen.sh` is ran again in the future?

Reply:
yes, auto generated. I am not totally sure what to do here. Maybe we should set grpcio to the lowest version that we support in the setup.cfg?

Reply:
Hmm, `grpcio==1.57.0` is not even a year old yet. Would it be possible to generate the stubs using the minimum version, which is `1.57.0` for now, and then bump this dependency at some point in the future? Perhaps we should fix the version of `grpcio` in `requirements-devel.txt` for this reason (but keep the range in `setup.cfg` for users)